docs/getting-started/publishing-keys.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+---
+
+# Key Publishing Guide
+
+This page explains how to configure the server and application to allow
+publishing keys.
+
+## Prerequisites
+
+If you haven't started developing a mobile app, please take a look at the
+[Android Exposure Notifications Reference App](https://github.com/google/exposure-notifications-android)
+
+You will need to
```

Comment:
Can you write this in the present tense (e.g. "You need to" instead of "You will need to")